adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/VariantContextRDD.scala

```diff
@@ -132,7 +132,7 @@ case class VariantContextRDD(rdd: RDD[VariantContext],
   /**
    * @return Returns a GenotypeRDD containing the Genotypes in this RDD.
    */
-  def toGenotypeRDD: GenotypeRDD = {
+  def toGenotypeRDD(): GenotypeRDD = {
```

Comment:
For correctness, I believe this also needs to send the `optPartitionMap` with it. It should have been doing that before.